apps/condo/domains/common/utils/helpers.ts

```diff
@@ -1,7 +1,25 @@
+import { ParsedUrlQuery } from 'querystring'
+import get from 'lodash/get'
+
 export const formatPhone = (phone?: string): string => {
     if (!phone) {
         return phone
     }
 
     return phone.replace(/(\d)(\d{3})(\d{3})(\d{2})(\d{2})/, '$1 ($2) $3-$4-$5')
-}
\ No newline at end of file
+}
+
+export const getFiltersFromQuery = (query: ParsedUrlQuery): {
```

Comment:
Add generic Here, please.
```suggestion
export const getFiltersFromQuery = <T>(query: ParsedUrlQuery): T | {} => {

```